specs/Deployment/DeploymentAPI.md

```diff
@@ -129,9 +131,57 @@ again.
     }
 ```
 
+### DeploymentInitializeOptions
+
+#### ForceDeployment
+
+When this API may install newer version of one or more of the WinAppSDK packages (i.e. an update)
+and if there are any currently running process(es) associated with these WinAppSDK package(s) (in
+other words, if any of the WinAppSDK packages to be updated are currently in use), then this API
+will fail installing that WinAppSDK package. But this update to the WinAppSDK packages can be forced
+by using DeploymentInitializeOptions object and setting ForceDeployment option before passing it to
+this API. This option when set will shutdown the application(s) associated with WinAppSDK packages,
```

Comment:
"This option when set will shutdown the application(s)..."  shut down should be two words in this instance. (verb form)